index.html

```diff
@@ -11,14 +11,15 @@
     <title>My Portfolio</title>
 </head>
 <body>
+  <div class="intro">
     <header>
       <div class="menu-popup">
         <nav class="nav-bar">
         <a href="#"><h1 class="logo"> My Logo</h1></a>
             <ul class="nav-links">
-                <li> <a href="#">Portfolio</a></li>
-                <li> <a href="#">About</a></li>
-                <li class="contact"> <a href="#">Contact</a></li>
+                <li> <a href="#Portfolio">Portfolio</a></li>
+                <li> <a href="#About">About</a></li>
+                <li class="contact"> <a href="#Contact">Contact</a></li>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_200](https://user-images.githubusercontent.com/69638013/169013706-aaef8ea9-7d44-49b6-bc89-8a192dc57936.png)  |  ![Screenshot_199](https://user-images.githubusercontent.com/69638013/169013515-394b8bde-c8b2-43eb-9708-a81d6dd35183.png)

- [ ] Kindly align the mobile nav links to the left 👍🏿 

- [ ] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 

### OPTIONAL

- [ ] It would be better for the background not to be scrollable when the mobile nav is open 👍🏿 


- On desktop:
![Screenshot_198](https://user-images.githubusercontent.com/69638013/169015773-30b46b27-1907-4264-805d-14f34f01bb50.png)

- [ ] Kindly remove the background color for the contact link, see screenshot above 👍🏿

Reply:
- [ ] As the last reviewer mentioned please make your background blurry, you can use `backdrop-filter` see this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) for more info.